src/framework/global/CMakeLists.txt

```diff
@@ -205,6 +208,7 @@ list(APPEND MODULE_INCLUDE
 list(APPEND MODULE_LINK
     ${FS_LIB}
     ${Z_LIB}
+    stduuid
```

Comment:
"A C++17 cross-platform single-header library implementation" - there shouldn't be a library there, just header file(s)

Reply:
I think is no need to add the cmake project of stduuid, we probably just need to install the necessary definitions before include

Reply:
I recently learned that `target_link_libraries` also transitively sets include directories for targets. So, in stduuid/CMakeLists.txt, we see 
```cmake
target_include_directories(${PROJECT_NAME} INTERFACE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
         $<INSTALL_INTERFACE:include>)
```
which means that all targets with `target_link_libraries(target stduuid)` automatically get `stduuid/include` added to their include directories. 

So for that reason, it may still make sense to do it like it's done now.

Reply:
but we known where placed includes and they static for us (not installed externally), so I think no need for us )